qa/rpc-tests/test_framework/script.py

```diff
@@ -226,7 +226,8 @@ def __new__(cls, n):
 
 # expansion
 OP_NOP1 = CScriptOp(0xb0)
-OP_NOP2 = CScriptOp(0xb1)
+OP_CHECKLOCKTIMEVERIFY = CScriptOp(0xb1)
+OP_NOP2 = OP_CHECKLOCKTIMEVERIFY
```

Comment:
I am not sure this "compatibility" should be retained. If someone is specifying OP_NOP2, they may actually be expecting OP_NOP2 behaviour. Better to error and let them explicitly change to OP_CHECKLOCKTIMEVERIFY.